spec/consensus/proposer-based-timestamp/pbts-sysmodel.md

```diff
@@ -298,7 +263,8 @@ If
 then there is a correct process `p` (not necessarily the same above considered) such that:
 
 - `beginRound(p,v.round) <= proposalReceptionTime(p,v.round) <= beginRound(p,v.round+1)` and
```

Comment:
Not sure if we need the left inequality. We may receive a proposal before entering the round it refers to. See comment above on line 142.